src/tests/Microsoft.PowerFx.Interpreter.Tests/MutationScripts/Collect.txt

```diff
@@ -69,3 +69,27 @@ Table({a:1,b:1},{a:2,b:2})
 
 >> Collect(partialT1, {a:3});Last(partialT1).b
 Blank()
+
+>> Set(t1, [1,2,3])
+Table({Value:1},{Value:2},{Value:3})
+
+>> Collect(t1, 4)
+{Value:4}
+
+>> Collect(t1, [5,6,7],8,9,10)
+Table({Value:5},{Value:6},{Value:7},{Value:8},{Value:9},{Value:10})
+
+>> 0;t1
+Table({Value:1},{Value:2},{Value:3},{Value:4},{Value:5},{Value:6},{Value:7},{Value:8},{Value:9},{Value:10})
+
+>> Collect(t1, Table({Value:11},Blank()))
+Table({Value:11})
+
+>> Collect(t1, Table({Value:12},Error({Kind:ErrorKind.Custom})))
+Error({Kind:ErrorKind.Custom})
```

Comment:
Another interesting test would be something like `Collect(t1, Table({Value:12}, Error(...), {Value:14}))` - to determine whether a value *after the error* is collected or not.

Reply:
I can add this test case. In the current code it wont be collected.